examples/src/test/scala/za/co/absa/atum/examples/SampleMeasurementsS3RunnerExampleSpec.scala

```diff
@@ -15,10 +15,12 @@
 
 package za.co.absa.atum.examples
 
+import org.scalatest.Ignore
 import org.scalatest.funsuite.AnyFunSuite
 import za.co.absa.atum.utils._
 
-class SampleMeasurementsS3RunnerSpec extends AnyFunSuite
+@Ignore
```

Comment:
Why is this here?

Reply:
Because unlike the hadoop-fs tests, these tests should not be run against actual S3. Thus, they now:

1. serve as an example
2. can be run manually, provided certain conditions are met (files exist on S3 inside a specified bucket, KEY ID is supplied, local `saml` profile is supplied)